lib/flatpak-game.lua

```diff
@@ -375,24 +377,28 @@ function save_manifest(metadata, enable_network)
 end
 
 function build_export()
+    print('Building repository for ' .. metadata.id)
```

Comment:
Passing it back to the caller would be nice indeed, but printing in random places in the library is not...